docker/README.md

```diff
@@ -92,14 +92,13 @@ accordingly.
 
 Ex. If your link name is "storage" for an Elasticsearch container:
 ```
-  ES_HOSTS=http://$STORAGE_PORT_9200_TCP_ADDR:9200
+  ZIPKIN_STORAGE_ES_CLUSTER_NODES=$STORAGE_PORT_9200_TCP_ADDR:9200
```

Comment:
Can we maintain both variables and deprecate the former? As far as I understand we want to keep retrocompatibility for ease migration.

Reply:
We could keep some consistent with before. But things like storage selector, it is very skywalking style, can't be aligned with true/false style.